### ISSUE TITLE
Hello, I have the following questions about this paper.

### ISSUE THREAD

**Issue by Kitzzaaa** [fabiocarrara/features-adversarial-det#1]
1.I wonder if only adversarial examples are needed when training the detector? Need an original image? Or do you need labels for adversarial examples?
2.Hello, I would like to know when ‘the Features extraction from ILSVRC'12 TRAIN dataset’, do you use all the training set images? How many images did you use, if not all the images in the training set?
3.I want to know what adversarial_stats.py is mainly used for？

**Comment by fabiocarrara** [comment]
Hi,
1. you need both adversarial and original samples.
2. we used the whole ILSVRC'12 TRAIN dataset for computing centroids/medoids, but you can use a smaller subset and approximate them.
3. It is used to compute some statistics of generated adversarial examples for generating the plots of the paper.

Check the paper for more details: https://openaccess.thecvf.com/content_ECCVW_2018/papers/11130/Carrara_Adversarial_examples_detection_in_features_distance_spaces_ECCVW_2018_paper.pdf
